packages/experiments/src/components/Pagination/Pagination.types.ts

```diff
@@ -0,0 +1,117 @@
+import { IStyle, ITheme } from '../../Styling';
+import { IStyleFunctionOrObject, IRefObject } from '../../Utilities';
+
+export interface IPagination {}
+
+export interface IPaginationProps {
+  /**
+   * Required: Selected index
+   */
+  selectedPageIndex: number;
+
+  /**
+   * label for the previous page button
+   */
+  previousLabel: string;
+
+  /**
+   * Required: label for the next page button
+   */
+  nextLabel: string;
```

Comment:
Same in here: why required prop `previousLabel` and `nextLabel`? Doesn't design have some defaults for that? I kinda like the ones you used in the Custom example but better talk to designers.